packages/lit-element/package.json

```diff
@@ -23,14 +24,13 @@
     "/ts3.4/"
   ],
   "scripts": {
-    "build": "tsc && rm -rf ./ts3.4 && downlevel-dts . ts3.4 && cp tsconfig.json ./ts3.4/",
+    "build": "tsc",
     "build:watch": "tsc --watch",
     "build:babel-test": "babel src/test/lib/decorators_test.ts --out-file test/lib/decorators-babel_test.js",
-    "gen-docs": "typedoc src/lit-element.ts src/lib/updating-element.ts",
-    "test": "npm run build && npm run build:babel-test && wct",
-    "quicktest": "wct -l chrome -p",
-    "checksize": "rollup -c ; rm lit-element.bundled.js",
-    "format": "find src test | grep '\\.js$\\|\\.ts$' | xargs clang-format --style=file -i",
+    "test": "wtr test/*test.js --node-resolve --root-dir ../../",
+    "test:watch": "wtr test/*test.js --node-resolve --root-dir ../../ --watch",
+    "checksize": "rollup -c ; cat lit-element.bundled.js | gzip -9 | wc -c ; rm lit-element.bundled.js",
```

Comment:
FYI, in lit-html I've taken to removing gzip from the command line here because rollup-plugin-filesize reports the gzipped size and by outputting the ungzipped bytes to full precision you can see some small changes.

Reply:
Done.